lib/orchestrator/application.rb

```diff
@@ -104,6 +104,16 @@ def deploy_versions(language:, version_slugs:)
       PlatformConnection.new.deploy_versions(language, version_slugs)
     end
 
+    def deployed_versions(language: )
+      deployed_versions = PlatformConnection.new.deployed_versions(language)
+      version_slugs = deployed_versions[:response][:status].
+          select {|_, value| value }.
+          keys.
+          map { |s| s.to_s.tr("git-", "") }
```

Comment:
Do we need to the `to_s`?

Reply:
`NoMethodError: undefined method `tr' for :"git-a2mwqw":Symbol`

Reply:
That's what I get when omitting the `to_s`

Reply:
Posh. I didn't expect us to use strings there. OK, thanks.